paddle/operators/activation_op.cc

```diff
@@ -69,6 +69,22 @@ class ReluOpMaker : public framework::OpProtoAndCheckerMaker {
   }
 };
 
+template <typename AttrType>
+class LeakyReluOpMaker : public framework::OpProtoAndCheckerMaker {
+ public:
+  LeakyReluOpMaker(framework::OpProto *proto,
+                   framework::OpAttrChecker *op_checker)
+      : OpProtoAndCheckerMaker(proto, op_checker) {
+    AddInput("X", "Input of LeakyRelu operator");
```

Comment:
It will be better if the comment can follow our name convention. 
[name_convention](https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/operators/name_convention.md)

Reply:
I see, I didn't follow it because all the other operators in this file also did not follow the convention.

Reply:
Yeah, in fact, not everyone is familiar with the name convention, even the reviewer. 
We want to use a better docs generator tool to show the comments. Please take a look at our competitor's operator catalog. The comment is self-complained clearly. https://caffe2.ai/docs/operators-catalogue.html#accumulate 
We'd better make a better user experience.